zips/draft-coinholder-directed-grants.md

```diff
@@ -0,0 +1,259 @@
+
+“ZEC” refers to the native currency of Zcash on Mainnet.
+
+"Cycle" refers to a single iteration of the quarterly process of grant submission,
+voting, and grant disbursement.
+
+
+# Abstract
+
+This ZIP defines how the Coinholder Grants Program established in ZIP 1016 [^zip-1016] operates. It specifies a retroactive funding model, Administrator responsibilities, shielded and transparent voting procedures, participation thresholds, funding prioritization, partial funding, and payment execution by Key-Holder Organizations.
+
+
+# Motivation
+
+ZIP 1015 [^zip-1015] introduced a temporary, non-direct funding model that directed 12% of block rewards to a protocol lockbox and 8% to Zcash Community Grants.
+
+ZIP 1016 extends this by defining the Community and Coinholder Funding Model, allocating 12% of block rewards to a Coinholder Grants Program and 8% to Zcash Community Grants through Zcash’s third halving.
+
+While ZIP 1016 establishes a funding stream for coinholder-directed grants, it does not define how proposals are submitted, reviewed, voted, prioritized, or paid. This ZIP provides that missing operational process so that the Coinholder Grants Program can function as intended.
+
+
+# Privacy Implications
+
+TBD
+
+
+# Requirements
+
+The design of this ZIP is constrained to align with the coinholder decision-making mechanisms established in ZIP 1016.
+
+The process described in this ZIP should describe how proposals are submitted, how voting is performed, how voting outcomes are determined, and how disbursements are made.
+
+The roles of the Administrator, Key-Holder Organizations, Voting Authorities, and coinholders are expected to be clearly defined and distinguishable.
+
+This ZIP should describe Key-Holder responsibilities for custody, security, and legal compliance in a clear and understandable manner.
+
+The design is constrained to remain compatible with ZIP 1016 and with the NU 6.1 funding specification that governs key custody.
+
+The process described in this ZIP is intended to operate on a quarterly cycle, with recurring rounds for proposal submission, review, voting, and disbursement.
+
+This ZIP should not modify other Zcash funding programs or governance structures.
+
+This ZIP must not require any changes to Zcash consensus.
+
+
+# Specification
+
+## Program Structure and Custody
+
+The Coinholder Grants Program SHALL be administered by an organization to be designated prior to activation, referred to as the Administrator.
+
+Key custody and the n-of-m multisignature scheme controlling the Coinholder-Controlled Fund MUST follow the NU 6.1 funding specification referenced by ZIP 1016. Key-Holder Organizations MUST execute disbursements according to NU 6.1 [^zip-0271] and ZIP 1016’s veto provisions [^zip-1016-veto-process].
```

Comment:
The first sentence is an overspecification. We can just rely on the fact that we are importing the definition of "Key-Holder Organizations" from ZIP 1016, and leave it to define how keys are managed.